examples/Exploring-different-models.ipynb

```diff
@@ -0,0 +1,802 @@
+{
```

Comment:
<p>After reading the following paras and the cells--and that's golden content--taking the time to...</p><ul><li>mention the schema</li><li>how the schema was created</li><li>how models uses the schema</li><li>showing <code>schema.column_names</code> (so it is less magical)</li></ul><p>...those steps are all very valuable to anyone who is not already familiar with the schema object.</p><p> </p><p>So, I think the heading should change, but I'm not sure what to suggest.  The possible words to use are "NVTabular", "Schema", "Standard".</p><p> </p><p>Maybe "Benefits of Preparing Data with NVTabular" or "NVTabular and a Data Schema: Benefits of Standardization"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/256/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>Sorry, not sure I understand what heading should change? <code>Training Recommender Models</code> ? We decided that with Even. I dont think excluding training rec models  from the  heading will be informative and correct.  It is what we do in the following cells: we are training the models with Merlin Models. May be I can change it as <code>Training Recommender Models with Merlin Models</code>.</p>